CHANGELOG.md

```diff
@@ -6,6 +6,16 @@
 
 - Fix missing key warning.
 
+## Next
+
+### InputStepper
+
+- New component `InputStepper` is a number input with step buttons on the sides
```

Comment:
Hm, not super important, but all inputs have been named `<Type>Input`.

Should we continue being consistent, or is it not important enough?